src/CLI/CLI.php

```diff
@@ -215,15 +219,16 @@ public static function setResource(string $name, callable $callback, array $inje
     /**
      * task-name --foo=test
      *
-     * @param array $args
+     * @param non-empty-array<string> $args
      * @throws Exception
-     * @return array
+     * @return array<string, mixed>
      */
     public function parse(array $args): array
     {
         \array_shift($args); // Remove script path from args
 
         if (isset($args[0])) {
+            /** @phpstan-ignore-next-line */
```

Comment:
can we make this work rather than add an ignore? Maybe this code can be rewritten as:

```php
        $command = \array_shift($args);

        if ($command == null) {
            throw new Exception('Missing command');
        }

        $this->command = $command;
```